datasets/emotion/emotion.py

```diff
@@ -33,17 +34,33 @@
 _TEST_DOWNLOAD_URL = "https://www.dropbox.com/s/ikkqxfdbdec3fuj/test.txt?dl=1"
 
 
+class EmotionConfig(datasets.BuilderConfig):
+    def __init__(self, **kwargs):
+        super(EmotionConfig, self).__init__(**kwargs)
+
+
 class Emotion(datasets.GeneratorBasedBuilder):
+
+    VERSION = datasets.Version("0.1.0")
+    BUILDER_CONFIGS = [
+        EmotionConfig(
+            name="emotion",
+            version=datasets.Version("1.0.0"),
+            description="Emotion is a dataset of English Twitter messages with six basic emotions: anger, fear, joy, love, sadness, and surprise.",
+        )
+    ]
+
     def _info(self):
         class_names = ["sadness", "joy", "love", "anger", "fear", "surprise"]
         return datasets.DatasetInfo(
             description=_DESCRIPTION,
             features=datasets.Features(
-                {"text": datasets.Value("string"), "label": datasets.ClassLabel(names=class_names)}
+                {"tweet": datasets.Value("string"), "emotion": datasets.ClassLabel(names=class_names)}
```

Comment:
I guess you did this change to test the column renaming step of the task preparation.
Though ultimately we'll probably not rename the features of this dataset to avoid breaking changes

Reply:
yes, this was just to show how it might look in practice - i'll revert the changes before the final review!